dashboard/sample_metadata_classifier.py

```diff
@@ -231,5 +231,14 @@ def interpret(project, papers, bits):
             #  admission.
             date="2020-01",
             collection="bronchoalveolar lavage fluid")
+    elif project in papers["Bohl 2022"]["projects"]:
+        sample, = bits
+        return sample, dict(
+                country="Cambodia",
+                city="Chbar Mon",
+                state="Kampong Speu",
+                # Patient samples collected between March 2019 and October 2020
```

Comment:
Don't have to do this now, but if the paper has sample-level dates you could add sample-level dates to `metadata.tsv`, and then it would be available in `bits` to include here.

Reply:
I looked but didn't see any sample dates included in [PRJNA681566](https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA681566). They do use different sequencers for different samples; some are done on the iSeq 100 and others on the NextSeq 550. Is that something you would put in the metadata?

Reply:
I don't think the dates are critical, if they are not easy to get (or absent).

I don't think it is worth including the sequencer type. That isn't something we have for any of the other studies and I think it probably isn't needed?